components/TourBar/TourBar.stories.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import TourBar from "./TourBar";
+import { LngLat } from "~/pages";
+
+export default {
+  component: TourBar,
+  title: "Map Components",
+};
+
+const tour = [
+  {
+    label: "start",
```

Comment:
why is this information hard coded in when we have it in contentful?

Reply:
The purpose of Storybook is to demo the UI component for project collaborators. It's common practice to use static mock data rather than making API calls to get real content. This keeps the story simple and also allows us to use the mock content to talk about the component itself.